### ISSUE TITLE
Rename IndexedView to IndexedSeqView

### ISSUE THREAD

**Issue by julienrf** [scala/collection-strawman#477]

**Comment by Ichoran** [comment]
As I mentioned in the comment, I would as soon change `IndexedSeqOps` to `IndexedOps`; I don't believe there's any other sensible way to get indexing (especially since "Indexed" doesn't just mean that you can find an index--you can do that with `LinearSeq` and `SortedSet` and stuff too--it really means that indexing is efficient and natural).

**Comment by szeiger** [comment]
I agree that `Seq` is redundant in principle but `Indexed` wouldn't make a good name to replace `IndexedSeq`. Since `IndexedSeqOps` is the ops trait for `IndexedSeq`, it should be called `IndexedSeqOps` and not `IndexedOps`. This is consistent with other ops traits.

I have a slight preference for `IndexedView` over `IndexedSeqView` because `Seq` is redundant and unimportant for the view. OTOH, you get a `MapView` from a `Map` and a `SeqView` from a `Seq` so it makes sense that you get an `IndexedSeqView` from an `IndexedSeq`.

**Comment by SethTisue** [comment]
I'd suggest favoring consistency here. One less thing to remember.
